doc/user/content/get-started/quickstart.md

```diff
@@ -12,19 +12,35 @@ aliases:
   - /install/
 ---
 
-Materialize is the Cloud Operational Data Store that delivers the speed of
-streaming with the ease of a data warehouse. With Materialize, organizations can
-use SQL to transform, deliver, and act on fast-changing data.
+{{% text-style %}}
 
-This quickstart will get you up and running in a few minutes and with no
-dependencies, so you can experience the superpowers of an operational data
-store first-hand:
+Materialize provides always-fresh results while also providing [strong
+consistency guarantees](/get-started/isolation-level/). In Materialize, both
+[indexes](/concepts/indexes/ "Indexes represents query results stored in memory
+within a cluster") and [materialized views](/concepts/views/#materialized-views)
+**incrementally update** results when Materialize ingests new data; i.e., work
+is performed on writes. Because work is performed on writes, reads from these
+objects return up-to-date results while being computationally <redb>free</redb>.
 
-* **Interactivity**: get immediate responses from indexed warehouse relations and derived results.
+In this quickstart, you will:
 
-* **Freshness**: watch results change immediately in response to *your* input changes.
+- Create and query various [views](/concepts/views/) on sample auction data. The
+  data is continually generated at 1 second intervals to mimic a data-intensive
+  workload.
 
-* **Consistency**: results are always correct; never even transiently wrong.
+- Create an [index](/concepts/indexes "Indexes represents query results stored
+  in memory within a cluster") to compute and store view results in memory. As
+  new auction data arrives, the index <redb>incrementally updates</redb> view
+  results instead of recalculating the results from scratch, making fresh
+  results immediately available for reads.
+
+- Create and query views to verify that Materialize always serves
+  <redb>consistent results</redb>.
+
+The quickstart uses the default `quickstart` cluster. In Materialize, a
```

Comment:
This line also feels a little out of place for me! Maybe it was meant to be alongside the other one?